debian/rules

```diff
@@ -101,6 +95,7 @@ endif
 	    -DPLUGIN_TOKUDB=NO \
 	    -DPLUGIN_CASSANDRA=NO \
 	    -DPLUGIN_AWS_KEY_MANAGEMENT=NO \
+	    -DPLUGIN_COLUMNSTORE=NO \
```

Comment:
How is was going to be built at all if we put $(CMAKEFLAGS) before -DPLUGIN_COLUMNSTORE=NO \ ?

Reply:
Yes, the native build should not build ColumnStore by design. This is now correct. To build CS the autobake-deb.sh removes this line, which is enough to trigger the build of CS (no expicit YES is needed).